### PR TITLE
[DOCS] Unraid and FAQ Doc Updates

### PR DESCRIPTION
**Pull request by Svagtlys**
Updated Unraid install instructions following support request on Discord and made minor changes to FAQ and Docker install page

### CODE REVIEW

File: docs/source/guides/install/unraid.rst [hunk 1]
```diff
@@ -1,3 +1,14 @@
 Unraid
 --------------
-You can install our :unraid:`unraid community apps <community/apps?q=ytdl-sub#r>` through the  `Unraid Community Apps plugin <https://unraid.net/community/apps>`_. Uses Docker under the hood.
\ No newline at end of file
+You can install our :unraid:`unraid community apps <community/apps?q=ytdl-sub#r>` through the  `Unraid Community Apps plugin <https://unraid.net/community/apps>`_.
+
+
+After starting, if you installed the ``ytdl-sub-gui`` app, the code-server will be running at http://localhost:8443 (replace ``localhost`` with the IP of the computer running Unraid if you aren't trying to access ``ytdl-sub`` on that computer). Open this page in a browser to access and interact with ``ytdl-sub``. 
+
+At this time, we are unsure how to access the headless ``ytdl-sub`` console using the user ``abc`` and not ``root``. If you use Unraid and can help us figure this out, please contact the code owner or join the Discord.
```
Comment (jmbannon) [inline docs/source/guides/install/unraid.rst:8]: You should be able to do this via logging in as root (using the console button), then using the switch-user command `su abc`

File: docs/source/guides/install/unraid.rst [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+
+.. warning:: 
+  Do **NOT** use the below option to access ``ytdl-sub``. Running from this option **WILL** break things.
```
Comment (jmbannon) [inline docs/source/guides/install/unraid.rst:11]: maybe change to "break things by modifying files as the root user"

Comment (Svagtlys) [reply docs/source/guides/install/unraid.rst:11]: Changed to recommended

File: docs/source/guides/install/unraid.rst (hunk 1 above)
Comment (jmbannon) [reply docs/source/guides/install/unraid.rst:8]: Okay it should be `su -l abc`

Comment (Svagtlys) [reply docs/source/guides/install/unraid.rst]: Latest push updated to include headless access of Unraid app using `su abc -s /bin/bash`

**Pull request merged**